src/clj/game/cards/identities.clj

```diff
@@ -738,6 +738,11 @@
                                                               " [Credits] from the rez of " (:title current-ice)))
                                   (swap! state update-in [:bonus] dissoc :cost))))}]}
 
+   "Nathaniel \"Gnat\" Hall: One-of-a-Kind"
+   {:events {:runner-turn-begins {:msg "gain 1 credit"
+                                  :req (req (> 3 (count (:hand runner))))
```

Comment:
We recently (#3442 evolving into #3513) changed how `:req` works with event handlers to more properly align with game rules. The net effect is that `:req` is for deciding whether the handler exists at all, notably for cards that read "If X, then when Y, do Z". If a card instead reads "When X, then if Y, do Z", we've moved the Y condition into the effect of the card.